modules/era/sql/era_droplist.sql

```diff
@@ -1213,11 +1214,11 @@ INSERT INTO `mob_droplist` VALUES (130,0,0,1000,1429,@VRARE); -- Black Mages Tes
 INSERT INTO `mob_droplist` VALUES (130,0,0,1000,4770,@VRARE); -- Scroll Of Stone Iv (Very Rare, 1%)
 
 -- ZoneID: 208 - Antican Princeps
+INSERT INTO `mob_droplist` VALUES (131,0,0,1000,16995,@VCOMMON); -- Piece Of Rotten Meat (Very Common, 24%)
 INSERT INTO `mob_droplist` VALUES (131,0,0,1000,1118,@COMMON);   -- Antican Pauldron (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (131,0,0,1000,16995,@COMMON);  -- Piece Of Rotten Meat (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (131,0,0,1000,1540,@UNCOMMON); -- Dhalmel Leather Missive (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (131,0,0,1000,644,@RARE);      -- Chunk Of Mythril Ore (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (131,0,0,1000,1054,@RARE);     -- Quicksand Coffer Key (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (131,0,0,1000,1540,@VRARE);    -- Dhalmel Leather Missive (Very Rare, 1%)
```

Comment:
I would say the Dhalmel Leather Missive is maybe 5% instead of 1%? I'll leave it up to you but its much more likely with a 3.6% drop are at 8k points would be a 5% drop instead of a 1% drop. TH3+ is also around 8% which seems consistent with 5% base.

http://ffxidb.com/zones/208/antican-princeps